examples/portfolio/src/pages/projects.astro

```diff
@@ -4,16 +4,9 @@ import Footer from '../components/Footer/index.jsx';
 import Nav from '../components/Nav/index.jsx';
 import PortfolioPreview from '../components/PortfolioPreview/index.jsx';
 
-let { collection } = Astro.props;
-export async function createCollection() {
-  return {
-    async data() {
-      const projects = Astro.fetchContent('./project/*.md');
-      projects.sort((a, b) => new Date(b.published_at) - new Date(a.published_at));
-      return projects.filter(({ published_at }) => !!published_at);
-    }
-  }
-}
+const projects = Astro.fetchContent('./project/*.md');
```

Comment:
```suggestion
const projects = Astro.fetchContent('./project/*.md')
```
It looks like there shouldn't be a semicolon there since it's filtered on the next line?